src/constants.ts

```diff
@@ -1,3 +1,14 @@
-export const GAS_LIMIT = 500_000;
+// The number of past blocks to lookup when fetching events.
 export const BLOCK_COUNT_HISTORY_LIMIT = 300;
+
+// The default amount of time before a "retryable" promise is timed out and retried
 export const DEFAULT_RETRY_TIMEOUT_MS = 5_000;
+
+// The Priority Fee in Wei
+export const PRIORITY_FEE = 3120000000;
```

Comment:
There was a [change recently](https://github.com/api3dao/airnode/pull/933/files#diff-09bfa05deaf24c684349763bbc069a051da261e1a972b10df5d6ca252abcf5e9) to rename this to `PRIORITY_FEE_IN_WEI`. We might want to replicate that here

```suggestion
export const PRIORITY_FEE_IN_WEI = 3_120_000_000;
```